pkg/hintrunner/zero/zerohint_math.go

```diff
@@ -501,3 +501,78 @@ func createSplitIntHinter(resolver hintReferenceResolver) (hinter.Hinter, error)
 	}
 	return newSplitIntHint(output, value, base, bound), nil
 }
+
+func newUnsignedDivRemCodeHinter(value, div, q, r hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "UnsignedDivRem",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.math_utils import assert_integer
+			//> assert_integer(ids.div)
+			//> assert 0 < ids.div <= PRIME // range_check_builtin.bound, \
+			//>     f'div={hex(ids.div)} is out of the valid range.'
+			//> ids.q, ids.r = divmod(ids.value, ids.div)
+
+			value, err := hinter.ResolveAsFelt(vm, value)
+			if err != nil {
+				return err
+			}
+			div, err := hinter.ResolveAsFelt(vm, div)
+			if err != nil {
+				return err
+			}
+
+			qAddr, err := q.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+			rAddr, err := r.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			// (PRIME // range_check_builtin.bound)
+			divUpperBound := new(big.Int).Div(fp.Modulus(), utils.Uint256Max128.ToBig())
```

Comment:
I think this should be just calculated once and stored as a constant somewhere. 

Something like: 
```
//> (PRIME // range_check_builtin.bound)
var divUpperBound big.Int
divUpperBound.SetString("10633823966279327296825105735305134080", 10)
```

But I'm not sure what it should be called.

Reply:
Do you know if it's used somewhere else in the code? If it's not, I would say to leave it just like this for now.

Reply:
Also I have a question here. Where did you get the value from  range_check_builtin.bound?
I've seen it being used other times and its value is 1<<128, and you can use `utils.FeltMax128` for it, but is not the same for `utils.Uint256Max128`, this one represents `(1<<128) -1`  if I'm not mistaken

Reply:
Yep you're right it should be 2**128. I'll be more careful.

Reply:
I just made it:

```
// (PRIME // range_check_builtin.bound)
// 800000000000011000000000000000000000000000000000000000000000001 // 2**128
var divUpperBound big.Int
divUpperBound.SetString("8000000000000110000000000000000", 16)
```